sklearn/notebooks/Online Prediction with scikit-learn.ipynb

```diff
@@ -381,18 +381,25 @@
     "Now it’s time to get your model online and ready for predictions. The model version requires a few components as specified [here](https://cloud.google.com/ml-engine/reference/rest/v1/projects.models.versions#Version).\n",
     "\n",
     "* __name__ - The name specified for the version when it was created. This will be the `VERSION_NAME` variable you declared at the beginning.\n",
-    "* __deployment Uri__ (curl) or __origin__ (gcloud) - The Google Cloud Storage location of the trained model used to create the version. This is the bucket that you uploaded the model to with your `BUCKET_ID`\n",
-    "* __runtime__ version - The Google Cloud ML runtime version to use for this deployment. This is set to 1.4\n",
+    "* __model__ - The name of the model container we created in Part 3. This is the `MODEL_NAME` variable you declared at the beginning.\n",
+    "* __origin__ (gcloud) or __deployment Uri__ (curl) - The Google Cloud Storage location of the trained model used to create the version. This is the bucket that you uploaded the model to with your `BUCKET_ID`\n",
+    "* __runtime version__ - [Select Google Cloud ML runtime version](https://cloud.google.com/ml-engine/docs/tensorflow/runtime-version-list) to use for this deployment. This is set to 1.4\n",
     "* __framework__ - The framework specifies if you are using: `TENSORFLOW`, `SCIKIT_LEARN`, `XGBOOST`. This is set to `SCIKIT_LEARN`\n",
     "* __pythonVersion__ - This specifies whether you’re using Python 2.7 or Python 3.5. The default value is set to `“2.7”`, if you are using Python 3.5, set the value to `“3.5”`\n",
     "\n",
-    "\n",
-    "Note: It can take several minutes for you model to be available.\n",
-    "\n",
     "Note: If you require a feature of scikit-learn that isn’t available in the publicly released version yet, you can specify “runtimeVersion”: “HEAD” instead, and that would get the latest version of scikit-learn available from the github repo. Otherwise the following versions will be used:\n",
     "* scikit-learn: 0.19.0"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we need to create a YAML file to configure our model version.\n",
+    "\n",
+    "__REPLACE:__ `true-ability-192918` with your `BUCKET_ID`\n"
```

Comment:
PREVIOUSLY_SPECIFIED_BUCKET_ID is what I'll be using, instead of true-ability-192918

Reply:
Updated